SA3/qa_web/views.py

```diff
@@ -384,3 +409,28 @@ def pagination_data(self, paginator, page, is_paginated):
 
         return context
 
+
+# edit post
+@login_required(login_url='/login/')
+def edit(request, id_):
```

Comment:
This block is repeated twice! My bad! Probably happened during conflict resolving. I will change this right now

Reply:
Actually looking at it, it seems that there was already two functions `edit(request,id_)` declared in `views.py`. It was just a coincidence that the one sending the correct context was the one further down, hence the one executed.